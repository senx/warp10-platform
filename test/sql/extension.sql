-- This file and its contents are licensed under the Apache License 2.0.
-- Please see the included NOTICE for copyright information and
-- LICENSE-APACHE for a copy of the license.

\c :TEST_DBNAME

--list all extension functions in public schema
SELECT DISTINCT proname
FROM pg_proc
WHERE OID IN (
    SELECT objid
    FROM pg_catalog.pg_depend                                                                                                               WHERE   refclassid = 'pg_catalog.pg_extension'::pg_catalog.regclass AND
    refobjid = (select oid from pg_extension where extname='timescaledb') AND
    deptype = 'e' and classid = 'pg_catalog.pg_proc'::regclass
) AND pronamespace = 'public'::regnamespace
ORDER BY proname;
