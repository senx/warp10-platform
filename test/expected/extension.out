-- This file and its contents are licensed under the Apache License 2.0.
-- Please see the included NOTICE for copyright information and
-- LICENSE-APACHE for a copy of the license.
\c :TEST_DBNAME
--list all extension functions in public schema
SELECT DISTINCT proname
FROM pg_proc
WHERE OID IN (
    SELECT objid
    FROM pg_catalog.pg_depend                                                                                                               WHERE   refclassid = 'pg_catalog.pg_extension'::pg_catalog.regclass AND
    refobjid = (select oid from pg_extension where extname='timescaledb') AND
    deptype = 'e' and classid = 'pg_catalog.pg_proc'::regclass
) AND pronamespace = 'public'::regnamespace
ORDER BY proname;
             proname              
----------------------------------
 add_compress_chunks_policy
 add_dimension
 add_drop_chunks_policy
 add_reorder_policy
 alter_job_schedule
 attach_tablespace
 chunk_relation_size
 chunk_relation_size_pretty
 compress_chunk
 create_hypertable
 decompress_chunk
 detach_tablespace
 detach_tablespaces
 drop_chunks
 first
 get_telemetry_report
 histogram
 hypertable_approximate_row_count
 hypertable_relation_size
 hypertable_relation_size_pretty
 indexes_relation_size
 indexes_relation_size_pretty
 interpolate
 last
 locf
 move_chunk
 remove_compress_chunks_policy
 remove_drop_chunks_policy
 remove_reorder_policy
 reorder_chunk
 set_adaptive_chunking
 set_chunk_time_interval
 set_integer_now_func
 set_number_partitions
 show_chunks
 show_tablespaces
 time_bucket
 time_bucket_gapfill
 timescaledb_post_restore
 timescaledb_pre_restore
(40 rows)

